modules/api/cmd/kubermatic-api/main.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 // Kubermatic Kubernetes Platform API
 //
 // This spec describes possible operations which can be made against the Kubermatic Kubernetes Platform API.
+// The API may introduce breaking changes and is not meant for external integrations.
```

Comment:
```suggestion
// Package docs Kubermatic Kubernetes Platform API
//
// This OpenAPI 2.0 specification describes the REST APIs used by the Kubermatic Kubernetes Platform Dashboard.
//
// KKP API is an **internal API** that is built primarily to complement the KKP dashboard. Our users are strongly advised to use Kubernetes APIs i.e. [KKP CRDs](https://docs.kubermatic.com/kubermatic/v2.22/references/crds/) to interact with and integrate KKP into their own platforms.
//
// Since KKP API is **not a public API**, users can expect breaking changes without any formal announcements.
```

Reply:
@pkprzekwas I was thinking of something like this. More detailed instead of just a one-liner. WDYT?

Reply:
Now I feel stupid with my oneliner 😅 . But this is exactly what we need! Thanks @ahmedwaleedmalik , let me update swagger.